core/bin/external_node/src/config/mod.rs

```diff
@@ -794,6 +1075,35 @@ impl ExperimentalENConfig {
     pub fn state_keeper_db_block_cache_capacity(&self) -> usize {
         self.state_keeper_db_block_cache_capacity_mb * BYTES_IN_MEGABYTE
     }
+
+    pub fn from_configs(general_config: &GeneralConfig) -> anyhow::Result<Self> {
```

Comment:
IMO, smaller config structs like `ExperimentalENConfig` can be removed immediately in favor of using `GeneralConfig` directly; not insisting though.

Reply:
I agree, i don't want to do this in this pr, just for making less mistakes